packages/aws-cdk-lib/cloud-assembly-schema/schema/cloud-assembly.schema.json

```diff
@@ -482,6 +482,11 @@
                 "props": {
                     "$ref": "#/definitions/ContextQueryProperties",
                     "description": "A set of provider-specific options."
+                },
+                "disableContextCaching": {
+                    "description": "Whether to disable persisting this to the context file.",
+                    "default": false,
+                    "type": "boolean"
```

Comment:
Curious about the schema changes, did you add these manually?

Reply:
Not manual, no. If I remember correctly this was the result of running `yarn update-schema` which I think was required for getting tests to pass.